### ISSUE TITLE
Google analytics isn't tracking searches done in CALC

### ISSUE THREAD

**Issue by toolness** [18F/calc#924]
So after installing the handy [Google Analytics Debugger extension](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en) that @jseppi recommended, I discovered that the single-page data explorer app isn't reporting searches to GA.  It's changing the page URL via `pushState`, which is great, but by default GA doesn't track those changes.

We can configure this using GA's guide on [Single Page Application Tracking](https://developers.google.com/analytics/devguides/collection/analyticsjs/single-page-applications).

However, it should also be noted that since the front-end is using the CALC API, which is served via api.data.gov, api.data.gov *should* be recording metrics on its usage.  So it's also likely that we can access the information we need via its analytics. I'm [asking how to do that on #g-analytics](https://gsa-tts.slack.com/archives/g-analytics/p1476373395000137) now.

**Comment by toolness** [comment]
I logged in to the [api.data.gov admin](https://api.data.gov/admin/) and it looks like it gives us access to raw logging data made to API endpoints, which we'd have to manually parse and generate our own statistics on--way more work than using Google Analytics.

Other than that, there are super high-level stats on the number of hits made to each endpoint, but there's only three major endpoints in the CALC API, so it's not super useful.

In other words, I think it'd be good to report more of the site usage via GA--even if some of that information is technically available through api.data.gov metrics, it'll be a lot easier to see the data and generate reports from it in GA.
